redhat-developer/app-services-cli
# feat(service-account): add support for service account SDK

Added a flag to login to control output format of service-account commands.

### Verification Steps

1. Run `rhoas service-account list -o json`. It should print output with control plane SDK (an object)
2. Login with the following flag:
```
 ./rhoas login --enable-auth-v2 
```
3.  Run `rhoas service-account list -o json` again. It should print output with service account SDK (an array)

### Type of change
